python/semantic_kernel/connectors/memory/redis/redis_collection.py

```diff
@@ -331,11 +467,27 @@ def _serialize_dicts_to_store_models(
     def _deserialize_store_models_to_dicts(
         self,
         records: Sequence[dict[str, Any]],
-        keys: Sequence[str],
+        keys: Sequence[str] | None = None,
```

Comment:
I am surprised that this didn't trigger a Mypy issue since the abstract method does not contain the `key` param.

Reply:
I think that's because of kwargs and it being optional, but is wasn't needed anymore because of another change, removed now.